dask/base.py

```diff
@@ -263,8 +263,8 @@ def _unpack(expr):
             typ = list if isinstance(expr, Iterator) else type(expr)
             if typ in (list, tuple, set):
                 tsk = (typ, [_unpack(i) for i in expr])
-            elif typ is dict:
-                tsk = (dict, [[_unpack(k), _unpack(v)]
+            elif typ in [dict, OrderedDict]:
```

Comment:
Slight preference to use a tuple here instead of a list (style nit). Otherwise this LGTM.